packages/ajax/stories/index.stories.mdx

```diff
@@ -4,7 +4,7 @@ import {
   html,
 } from '@open-wc/demoing-storybook';
 
-import { ajax } from '../src/ajax.js';
+import { ajax } from '../ajax.js';
```

Comment:
This seems like the wrong path, because ajax.js is indeed inside ../src/

Reply:
good catch 👍 fixed 💪